state/state.go

```diff
@@ -8,14 +8,18 @@ import (
 	"github.com/pganalyze/collector/config"
 )
 
+type DBStats struct {
+	RelationStats PostgresRelationStatsMap
+	IndexStats    PostgresIndexStatsMap
+	FunctionStats PostgresFunctionStatsMap
+}
+
 // PersistedState - State thats kept across collector runs to be used for diffs
 type PersistedState struct {
 	CollectedAt time.Time
 
 	StatementStats PostgresStatementStatsMap
-	RelationStats  PostgresRelationStatsMap
-	IndexStats     PostgresIndexStatsMap
-	FunctionStats  PostgresFunctionStatsMap
+	DBStats        map[Oid]*DBStats
```

Comment:
We should bump the `StateOnDiskFormatVersion` further down in this file, so if an old state file is read it gets ignored.

Reply:
Interesting--I had wondered about state compatibility but I tried running it with a statefile from master specifically to test that and did not have any problems so I figured it Just Works™ somehow. But versioning makes sense; I'll bump it.